<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>barexportdialog</class>
 <widget class="QDialog" name="barexportdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="saveFileLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Save file...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="savedLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>File not yet saved...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>62</y>
     <width>192</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="fileNameLabel">
        <property name="text">
         <string>File name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileNameLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="fileTypeLabel">
        <property name="text">
         <string>File type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fileTypeComboBox">
        <item>
         <property name="text">
          <string>.wav</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.mp3</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>saveFileLabel</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>savedLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
